mods/ra/weapons/missiles.yaml

```diff
@@ -434,24 +304,33 @@ SubMissile:
 		ImpactSounds: splash9.aud
 		ValidImpactTypes: Water
 
+SubMissile:
+	Inherits: ^SubMissileDefault
+	Range: 16c0
+	-Projectile:
+	Projectile: Bullet
```

Comment:
You can ignore the other comments concerning the next few lines as well I guess.